dojo/settings/settings.dist.py

```diff
@@ -463,6 +463,30 @@ def generate_url(scheme, double_slashes, user, password, host, port, path, param
 
 LEGACY_API_V1_ENABLE = env('DD_LEGACY_API_V1_ENABLE')
 
+AUTH_PASSWORD_VALIDATORS = [
```

Comment:
@blakeaowens This is the first time DefectDojo enforces password rules, hooray. We should have some form of documentation about that so users are not confused, either in our GitHub pages docs or in the dialogues where users can set a password.